app/components/notification-viewer.cjsx

```diff
@@ -1,53 +0,0 @@
-React = require 'react'
```

Comment:
I'm removing the popups for now to reduce confusion, but they may reappear when we get to it.